template-helpers/hbs-helpers.js

```diff
@@ -162,6 +162,16 @@ function breachMath(lValue, operator = null, rValue = null) {
 }
 
 
+function IntersectionObserverPolyfill(args) {
+  const topLevel = (args.data.exphbs.view === "top-level-page");
+  if (topLevel) {
+    /* eslint-disable-next-line quotes */
+    return `<script type="text/javascript" src="/dist/intersection-observer.min.js"></script>`;
+  }
+}
```

Comment:
Does this need any other `return` statement?

I see we have this below (views/partials/imports.hbs#9), but wanted to make sure we don't get into a weird case where we aren't trying to write `null` or `undefined` into the DOM:

```hbs
{{{ IntersectionObserverPolyfill }}}
```